_hooks/pre-commit

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+#
+
+# A git hook script that checks for trailing whitespace
+# in your commits. Bypass it with the --no-verify option
+# to git-commit
+#
+# usage: make a soft link to this file,
+# e.g., ln -s ~/config/pre-commit.git.sh ~/some_project/.git/hooks/pre-commit
+
+if test "$(git diff --check --cached)"
```

Comment:
I'm not sure I understand what you mean here?  Do you mean to incorporate this hook into Travis as well and fail if it finds whitespace?